features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -644,6 +645,16 @@ internal class DatadogRumMonitor(
         handleEvent(RumRawEvent.TelemetryEventWrapper(telemetryEvent))
     }
 
+    override fun enableJankStatsTracking(activity: Activity) {
+        val rumFeatureScope = sdkCore.getFeature(Feature.RUM_FEATURE_NAME)
+            ?.unwrap<RumFeature>()
+        if (rumFeatureScope == null) {
+            return
+        }
+
+        rumFeatureScope.enableJankStatsTracking(activity)
```

Comment:
minor improvement:

```suggestion
        sdkCore.getFeature(Feature.RUM_FEATURE_NAME)
            ?.unwrap<RumFeature>()
            ?.enableJankStatsTracking(activity)
```

but can be kept like that as well, for the easier debugging (although

Reply:
Nope I like yours better.